site/en/tutorials/structured_data/imbalanced_data.ipynb

```diff
@@ -425,14 +466,21 @@
         ">   $\\frac{\\text{true positives}}{\\text{true positives + false positives}}$\n",
         "*   **Recall** is the percentage of **actual** positives that were correctly classified\n",
         ">   $\\frac{\\text{true positives}}{\\text{true positives + false negatives}}$\n",
+        "\n",
+        "**Note:** Accuracy is not a helpful metric for this task. You can have 99.8%+ accuracy on this task by predicting False all the time.  \n",
+        "\n",
+        "#### Other metrices\n",
+        "\n",
+        "The following metrics take into account all possible choices of thresholds $t$.\n",
+        "\n",
         "*   **AUC** refers to the Area Under the Curve of a Receiver Operating Characteristic curve (ROC-AUC). This metric is equal to the probability that a classifier will rank a random positive sample higher than a random negative sample.\n",
         "*   **AUPRC** refers to Area Under the Curve of the Precision-Recall Curve. This metric computes precision-recall pairs for different probability thresholds. \n",
         "\n",
-        "Note: Accuracy is not a helpful metric for this task. You can have 99.8%+ accuracy on this task by predicting False all the time.  \n",
         "\n",
-        "Read more:\n",
-        "*  [True vs. False and Positive vs. Negative](https://developers.google.com/machine-learning/crash-course/classification/true-false-positive-negative)\n",
-        "*  [Accuracy](https://developers.google.com/machine-learning/crash-course/classification/accuracy)\n",
+        "#### Read more:\n",
+        "*   [Strictly Proper Scoring Rules, Prediction, and Estimation](www.stat.washington.edu/people/raftery/Research/PDF/Gneiting2007jasa.pdf)\n",
```

Comment:
Include the protocol in the URL, or at least start with `//`. Otherwise this is interpreted as a relative path.

Reply:
```suggestion
        "*   [Strictly Proper Scoring Rules, Prediction, and Estimation](https://www.stat.washington.edu/people/raftery/Research/PDF/Gneiting2007jasa.pdf)\n",
```